src/SFA.DAS.AdminService.Web/Controllers/Roatp/Apply/RoatpFinancialController.cs

```diff
@@ -153,36 +158,32 @@ public async Task<IActionResult> DownloadFiles(Guid applicationId, Guid sectionI
         [HttpPost("/Roatp/Financial")]
         public async Task<IActionResult> Grade(FinancialApplicationViewModel vm)
         {
-            var application = await _applyApiClient.GetApplication(vm.Id);
+            var application = await _applyApiClient.GetApplication(vm.ApplicationId);
             if (application is null)
             {
                 return RedirectToAction(nameof(OpenApplications));
             }
 
             if (ModelState.IsValid)
             {
-                // replace with reference to RoATP sections
-
-                //var financialSequence = await _qnaApiClient.GetSequenceBySequenceNo(application.ApplicationId, FINANCIAL_SEQUENCE_NO);
-                //var financialSection = await _qnaApiClient.GetSectionBySectionNo(application.ApplicationId, FINANCIAL_SEQUENCE_NO, FINANCIAL_SECTION_NO);
-
-                var grade = new FinancialGrade
+                var financialReviewDetails = new FinancialReviewDetails
```

Comment:
Personally I would have left this as FinancialGrade so it's synonymous with EPAO

Reply:
Yeah, I didn't like FinancialGrade as a name for an entity that contains assessment grade, date graded, and/or comments, financials due date

Reply:
Might be worth referring this one to Alan to see if he has any preference from an Architecture perspective